### PR TITLE
Change `DeviceSpec` to `Device`

### PR DESCRIPTION
**Pull request by thisac**
**Description of the Change:**
Strawberry Fields v0.22.0 has been released with a breaking change, renaming the `DeviceSpec` class to `Device` and the `device_spec` engine method to `device`.

* Changes instances of `sf.DeviceSpec` to `sf.Device`.
* Changes calls to `sf.RemoteEngine.device_spec` to `sf.RemoteEngine.device`.

**Benefits:**
Things work as they should with the new Strawberry Fields release.

**Possible Drawbacks:**
None

**Related GitHub Issues:**
None

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,9 @@
 
 ### Bug fixes
 
+* Changed `DeviceSpec` to `Device` due to a breaking change in Strawberry Fields.
+  [(#)](https://github.com/PennyLaneAI/pennylane-sf/pull/)
+
 ### Contributors
```
Comment (josh146) [inline .github/CHANGELOG.md:17]: don't forget to add your name down here!

**Review by josh146 — APPROVED** [review]
Thanks @thisac! Happy for this to be merged if all test pass.

Should we make a bugfix release?

**Review by antalszava — APPROVED** [review]
Looks good :100: :slightly_smiling_face:

File: setup.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 
 requirements = [
-    "strawberryfields>=0.20",
+    "strawberryfields>=0.22",
```
Comment (antalszava) [inline setup.py:25]: Pinned here too :+1:

**Pull request merged**